src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -2685,12 +2815,105 @@ export function generateSamplePointsCube(
   // Win 11, NVidia 2070 Super: 16
   // Linux, AMD Radeon Pro WX 3200: 256
   // MacOS, M1 Mac: 256
+  //
+  // Note: When doing `textureGather...` we can't use texel centers
```

Comment:
Just wanted to say that this section was really informative and helpful in explaining the issues with cube map edges!

Reply:
ty